docs/low-code-jobs/databricks-jobs.md

```diff
@@ -3,23 +3,165 @@ sidebar_position: 1
+| Scheduler                 | The underlying engine that's going to execute your job. Databricks is recommended.                                                                                                                                                                  |
+| Fabric                    | The [execution fabric](/concepts/fabric) to which the job is going to be deployed.                                                                                                                                                                  |
+| Cluster Size              | The [default size](/concepts/fabric#whats-in-a-fabric) of the cluster, that's going to be created for the job to run.                                                                                                                               |
+| Cluster Mode              | Can be selected as `Single` (all Gems within job run on single cluster) or `Multi` (all Gems within job run on separate new cluster)                                                                                                                |
+| Schedule Interval         | Defines how often your job is going to run. The interval is defined using the [Quartz format](http://www.quartz-scheduler.org/documentation/quartz-2.3.0/tutorials/crontrigger.html). You can click on the clock icon, to easily pick the interval. |
+| Pipeline level Timeout    | Timeout at Pipeline level                                                                                                                                                                                                                           |
+| Alerts Email for Pipeline | Comma separated list of emails, that are going to receive notifications on specific job status events (job start, failure, or success) for entire Pipeline.                                                                                         |
+| Per Gem Timeout           | Timeout for each Gem in job Pipeline                                                                                                                                                                                                                |
+| Number of retries per Gem | Number of retries for each Gem in job Pipeline                                                                                                                                                                                                      |
+| Alerts Email per Gem      | Comma separated list of emails, that are going to receive notifications on specific job status events (job start, failure, or success) for each Gem in job Pipeline.                                                                                |
+
 ## Deployment
 
+To deploy a job on Databricks, we need to release the project from Prophecy UI as shown in example below. As soon as the project is
```

Comment:
job -> Job.

Might be worth linking to the "Release" section on the Project page to save duplicate documentation.